### PR TITLE
Added a break statement to fix layout issues of 'use original' link

### LINKED ISSUE #3835: "use original" profile photo link is misaligned
### Describe the problem and steps to reproduce it:
The **use original** link when you have set a profile photo but not yet submitted is misaligned.

1. Click **Choose Photo...**
2. Upload a photo
3. The **use original** link appears

### What happened?

The **use original** link is misaligned and incorrectly capitalised.

### What did you expect to happen?

The **use original** link should be under the **Choose Photo...** button like the **Delete current photo** link.

The text of the link should start with an uppercase letter as well **Use original** not **use original**.

### Anything else we should know?
screenshot 2016-10-21 13 08 13
> **use original** link

screenshot 2016-10-21 13 12 28
> **Delete current photo** link

### PR DESCRIPTION
**Pull request by vijeth-aradhya**
Fixes #3835 

I have added the `<br>` statement just above the `<div class='invisible-upload'>` to fix this. 

May be there's better way to do it. I could not understand how the 'use original' came up in the code to modify that directly. Hence, I have made the above changes.

Please let me know if there's a better way to fix it.

### CODE REVIEW

File: src/olympia/users/templates/users/edit.html [hunk 1]
```diff
@@ -92,6 +92,7 @@
               <img src="{{ amouser.picture_url }}"
                    alt="{% if not amouser.picture_type %}{{ _('No Photo') }}{% endif %}"
                    class="avatar photo">
+              <br>
```
Comment (tofumatt) [inline src/olympia/users/templates/users/edit.html]: You should be able to add `clear: both` to `.invisible-upload` in CSS to get it to clear without adding the line break.

**Comment by muffinresearch** [comment]
The `.use-original` link appears to get a style property added via JS. If you look that up you might be able to change that to set `display: block` instead since that is probably what's used as the toggle to display it.

**Comment by vijeth-aradhya** [comment]
@muffinresearch thank you! I found the JS code which was setting the wrong display property and corrected it :)

**Comment by mstriemer** [comment]
This looks much better, thanks!

Can you make a couple small tweaks while you're in there?

1. On line 33 when `$a` is assigned update the string to get `gettext('Use original')` (capitalise **use**)
2. On line 43 update it so that the link is added to the bottom of the container rather than after the photo. You should be able to use something like `$avatar.closest('li').append($a)` to do this

**Comment by vijeth-aradhya** [comment]
@mstriemer I have made the exact changes that you have asked for. Is there anything else?

**Comment by mstriemer** [comment]
Perfect! Thanks for the patch @vijeth-aradhya!

**Pull request merged**